deployment/migration-assistant-solution/lib/solutions-stack.ts

```diff
@@ -132,8 +161,37 @@ export class SolutionsInfrastructureStack extends Stack {
             role: bootstrapRole
         })
 
+        let vpc;
+        if (props.createVPC) {
+            vpc = new Vpc(this, 'Vpc', {});
+        }
+        else {
+            const vpcIdParameter = new CfnParameter(this, 'VPCId', {
+                type: 'AWS::EC2::VPC::Id',
+                description: 'Specify the VPC id to place Migration resources into'
+            });
+            addParameterLabel(parameterLabels, vpcIdParameter, "VPC Id")
+
+            const availabilityZonesParameter = new CfnParameter(this, 'VPCAvailabilityZones', {
+                type: 'List<AWS::EC2::AvailabilityZone::Name>',
+                description: 'Availability Zone names in the selected VPC to use. Must be the same NUMBER and ORDER of the selected Private Subnet Ids.'
```

Comment:
"Select Availability Zones in the selected VPC. Please provide two zones at least, corresponding with the private subnets selected next."